### ISSUE TITLE
Decoding not worked

### ISSUE THREAD

**Issue by wqking1986** [ChargePoint/wireshark-v2g#48]
Hello, i have built done the wireshark in Linux Ubuntu, but it cant work, as the follow pictures, there is no "Vehicle to Grid Transfer Protocol" tab, please  show me the guide, and tell me why? Please help me!

![image](https://github.com/ChargePoint/wireshark-v2g/assets/11363041/b2f201aa-f28b-4410-b1f7-46a594f7aaed)

<img width="475" alt="1692614716343" src="https://github.com/ChargePoint/wireshark-v2g/assets/11363041/16f683a7-11e7-4c64-bbe4-237da8568eaa">

**Comment by chardin-cpi** [comment]
The v2g.lua is missing - we've had trouble and it needs to be copied into a location that is known. When we tested this in the README.md we found that this was the location to get it to work.

```
For Linux
- copy v2g.lua to `~/.local/lib/wireshark/plugins`
- copy v2gexi.so to `~/.local/lib/wireshark/plugins/4.0/epan`
```

So, I've heard people tell me that have copied to the v2g.lua to the same directory as the v2gexi.so and had that work - but, you will want to ensure you see the v2g.lua in the plugins to enable the decode to work - it needs both.

cc: @pthompson-cpi
